### ISSUE TITLE
Reported as not accessible by screen readers

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#69]
As reported here: https://twitter.com/serrebi/status/828626787437408257

![image](https://cloud.githubusercontent.com/assets/6483057/22654797/8cad8f2c-ec53-11e6-9c7a-6ae6f299d4de.png)

**Comment by collinbarrett** [comment]
FYI, until this is solved (and no other bugs/issues on the new site are outstanding), the old site is still live at https://v1.filterlists.com. Note that its database is no longer being update with no lists, etc.

**Comment by collinbarrett** [comment]
This tool could be helpful: http://wave.webaim.org/

**Comment by collinbarrett** [comment]
Further feedback from Brandon:
![image](https://cloud.githubusercontent.com/assets/6483057/22659474/5ae0a29e-ec63-11e6-90a0-b22d24ca7138.png)
![image](https://cloud.githubusercontent.com/assets/6483057/22659491/69d9e1ca-ec63-11e6-9dd8-60446053faac.png)

**Comment by collinbarrett** [comment]
Won't fix on current site, but will be sure to implement properly on v3 currently under construction.
